sympy/vector/operators.py

```diff
@@ -0,0 +1,182 @@
+from sympy.core import Expr
+from sympy.vector.coordsysrect import CoordSysCartesian
+from sympy.vector.vector import Vector
+from sympy.vector.deloperator import Del
+
+
+def _get_coord_sys_from_expr(expr):
+    return list(expr.atoms(CoordSysCartesian))[0]
+
+
+class Gradient(Expr):
+    """
+    Represents unevaluated Gradient.
+
+    Examples
+    ========
+
+    >>> from sympy.vector import CoordSysCartesian, Gradient
+    >>> R = CoordSysCartesian('R')
+    >>> s = R.x*R.y*R.z
+    >>> Gradient(s, R)
+    Gradient(R.x*R.y*R.z, R)
+
+    """
+
+    def __new__(cls, scalar):
+        obj = Expr.__new__(cls, scalar)
+        obj._scalar = scalar
+        return obj
+
+    def gradient(self):
```

Comment:
We already have the function in the global namespace, we don't need a repetition.